### DISCUSSION TITLE
G14 Custom Fan Curve Turns Off GPU Fan

### DISCUSSION THREAD

**Discussion by Zhang10010** [seerge/g-helper#1818] [Q&A] [answered]
After I turns on custom fan curve, GPU fan stops working and goes to 0 rpm while CPU fan works fine. This is not what it's supposed to do, right? Is there a way to fix this?
![image](https://github.com/seerge/g-helper/assets/55727338/6b047e60-4f9f-46f9-85b2-383fbf945302)
![image](https://github.com/seerge/g-helper/assets/55727338/916c006d-f788-434d-8fb9-ccdedac85315)

**Comment by seerge** [comment, marked as answer]
G15 has buggy bios that sometimes does things like that. Try to reboot, it may help :)
